python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -41,6 +41,110 @@
 __all__ = ["InsertFakesConfig", "InsertFakesTask"]
 
 
+def _add_fake_sources(image, objects, calibFluxRadius=12.0, logger=None):
```

Comment:
Top level comment: as we've decided we want to move away from use of the term 'fake', this should probably be changed in the future (e.g., to 'synthetic'). This is probably best addressed in future tickets such as DM-29628 however, so happy to leave as-is for now.